src/views/v2/rsvp/form/title.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Block: RSVP
+ * Form title
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/rsvp/form/title.php
+ *
+ * See more documentation about our Blocks Editor templating system.
+ *
+ * @link {INSERT_ARTICLE_LINK_HERE}
+ *
+ * @since TBD
+ *
+ * @version TBD
+ */
+
+$going = $this->get( 'going' );
+
+if ( 'going' === $going ) {
+	$this->template( 'v2/rsvp/form/going/title', [ 'rsvp' => $rsvp ] );
+} else {
+	$this->template( 'v2/rsvp/form/not-going/title', [ 'rsvp' => $rsvp ] );
+}
```

Comment:
Since the markup is the same for both these, why not just have the same markup and just change the text within it?

Reply:
I felt this was the type of template that could be overridden many times. The string is static, and if we do the logic here, we leave the template just for the text, and easier to customize for people that doesn't know what an if/else is. And that way it's only HTML for them.